src/main/app/src/app/core/actionbar/action-bar.component.ts

```diff
@@ -50,5 +52,9 @@ export class ActionBarComponent implements AfterViewInit {
                 this.matBottomSheetRef = this.bottomSheet.open(this.template, {closeOnNavigation: false, hasBackdrop: false});
             }
         });
+        this.disableActionBar$ = this.utilService.disableActionBar$.asObservable();
+        this.disableActionBar$.subscribe(state => {
```

Comment:
Moet die subscription niet ook weer verwijderd worden? Ik weet niet of dat moet maar geef het alleen als mogelijk puntje van aandacht.

Reply:
Nee deze hoeft niet verwijderd te worden